inst/example_workflow.R

```diff
@@ -0,0 +1,55 @@
+#file with example workflow for basic package functionality - will be helpful later tu build vignettes
+
+git_stats <- GitStats$new()
+
+git_stats$set_connection(
+  api_url = "https://api.github.com",
+  token = Sys.getenv("GITHUB_PAT"),
+  owners_groups = c("openpharma", "r-world-devs")
+)
+
+git_stats
+
+# examples for getting repos
+
+git_stats$get_repos_by_owner_or_group()
+
+git_stats$get_repos_by_codephrase("Dashboard")
+
+git_stats$get_repos_by_codephrase("Dashboard",
+                                  language = "Python")
+
+# set your team members
+
+git_stats$set_team(team_name = "RWD",
+                   "galachad",
+                   "krystian8207",
+                   "kalimu",
+                   "marcinkowskak",
+                   "Cotau",
+                   "maciekbanas")
+
+git_stats$get_repos_by_team("RWD")
+
+# you can plot repos sorted by last activity
+
+git_stats$plot_repos()
+
+# choose more repos to show
+
+git_stats$plot_repos(repos_n = 20)
+
+# and reset your repos show and plot it once more
+
+git_stats$get_repos_by_owner_or_group()$plot_repos(repos_n = 15)
+
+# examples for getting commits
+# still based on GraphQL for GitHub with poor performance
+
+git_stats$get_commits_by_team("RWD", date_from = "2022-01-01")$plot_commits(stats_by = "day")
+
+# may occur fatal Bad Gateway error - one of reasons to deprecate GraphQL in GitHub class
+
+git_stats$get_commits_by_team("RWD", date_from = "2020-01-01")$plot_commits(stats_by = "month")
+
+git_stats$get_commits_by_owner_or_group(date_from = "2022-01-01")$plot_commits(stats_by = "month")
```

Comment:
Bad Gateway error as noted:
`Bad Gateway (HTTP 502)
Github (openpharma): pulling commits from 37 repositories.                                  
Github (r-world-devs): pulling commits from 5 repositories.
Error in `[.data.table`(x, i, , ) : 
  Column 5 ['authors'] is a data.frame or data.table; malformed data.table.
In addition: Warning messages:
1: In setDT(ans) :
  Some columns are a multi-column type (such as a matrix column): [5]. setDT will retain these columns as-is but subsequent operations like grouping and joining may fail. Please consider as.data.table() instead which will create a new column for each embedded column.
2: In setDT(ans) :
  Some columns are a multi-column type (such as a matrix column): [5]. setDT will retain these columns as-is but subsequent operations like grouping and joining may fail. Please consider as.data.table() instead which will create a new column for each embedded column.
3: In setDT(ans) :
  Some columns are a multi-column type (such as a matrix column): [5]. setDT will retain these columns as-is but subsequent operations like grouping and joining may fail. Please consider as.data.table() instead which will create a new column for each embedded column.
4: In setDT(ans) :`